pkg/apply/event/event.go

```diff
@@ -286,15 +284,13 @@ type DeleteEvent struct {
 	Identifier object.ObjMetadata
 	Operation  DeleteEventOperation
 	Object     *unstructured.Unstructured
-	// If delete is skipped, this reason string explains why
-	Reason string
```

Comment:
I don't know how DeleteEvent ended up with a `Reason` field while PruneEvent and ApplyEvent don't, but it's "fixed" now!

Reply:
Nvm, the PruneEvent has it too.